spec/fixtures/proofing/aamva/requests/verification_request.xml

```diff
@@ -31,7 +27,8 @@
           <nc:LocationStateUsPostalServiceCode>VA</nc:LocationStateUsPostalServiceCode>
           <nc:LocationPostalCode>20176</nc:LocationPostalCode>
         </aa:Address>
-      <aa:DocumentCategoryCode>1</aa:DocumentCategoryCode></dldv:verifyDriverLicenseDataRequest>
+        <aa:DocumentCategoryCode>1</aa:DocumentCategoryCode>
+      </dldv:verifyDriverLicenseDataRequest>
```

Comment:
This is a good diff. 👏 This is the part I wanted to change in the fixture but couldn't or everything would break.

It's probably not worth the rest of this PR, though.